### PR TITLE
Don't log spammy input events to debug log

### PR DESCRIPTION
**Pull request by peppy**
Fixes on-screen log spam that is mostly useless.

2020-02-04 at 18 37 15

### CODE REVIEW

File: osu.Framework/Input/InputManager.cs [hunk 1]
```diff
@@ -558,6 +558,22 @@ protected virtual bool PropagateBlockableEvent(IEnumerable<Drawable> drawables,
...
         }
 
+        private bool shouldLog(UIEvent eventType)
+        {
+            switch (eventType)
+            {
+                case KeyDownEvent k:
+                    return !k.Repeat;
+
+                case DragEvent _:
```
Comment (smoogipoo) [inline osu.Framework/Input/InputManager.cs]: Allow at the very least `DragStartEvent`

File: osu.Framework/Input/InputManager.cs [hunk 2]
```diff
@@ -558,6 +558,22 @@ protected virtual bool PropagateBlockableEvent(IEnumerable<Drawable> drawables,
...
+                case MouseMoveEvent _:
```
Comment (smoogipoo) [inline osu.Framework/Input/InputManager.cs]: What about `ScrollEvent`? Using any sort of smooth scrolling is guaranteed to blast you with logs.

File: osu.Framework/Input/InputManager.cs (hunk 1 above)
Comment (peppy) [reply osu.Framework/Input/InputManager.cs]: DragStart doesn't derive DragEvent, so should be good

File: osu.Framework/Input/InputManager.cs (hunk 2 above)
Comment (peppy) [reply osu.Framework/Input/InputManager.cs]: oops, missed this one when i inversed the function.

**Pull request merged**